awx/main/models/unified_jobs.py

```diff
@@ -149,6 +149,10 @@ class Meta:
         default='ok',
         editable=False,
     )
+    credentials = models.ManyToManyField(
+        'Credential',
+        related_name='%(class)ss',
+    )
```

Comment:
`blank=True`? Though not sure that it matters in the way we're using the serializers...